src/index.js

```diff
@@ -46,13 +47,11 @@ module.exports = function (context) {
             tagName: 'script',
             innerHTML: `
               var _paq = window._paq = window._paq || [];
-              _paq.push(['setRequestMethod', 'POST']);
               _paq.push(['trackPageView']);
               _paq.push(['enableLinkTracking']);
```

Comment:
Hi is this PR still in development? I would need this change too.
Also not sure why this shouldn't work or be different from the current state. Maybe the order of the element creates the problem? My guide mentioned that my option should be added at the beginning for example.

So maybe we could make the following changes and add everything to the config? That way everyone can specify the order of elements like this in the config
```
matomo: {
  matomoUrl: 'https://vega.dev.trackmage.com/',
  siteId: '2',
  phpLoader: 'mm.php',
  jsLoader: 'mjs.php',
  tracking: [
    ['setRequestMethod', 'POST'],
    ['trackPageView'],
    ['enableLinkTracking']
    ['enableHeartBeatTimer'],
  ]
},
```

```suggestion
```